lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnDriverConfiguration.java

```diff
@@ -50,6 +51,10 @@
    * @see org.apache.reef.runtime.yarn.driver.parameters.YarnHeartbeatPeriod.class
    */
   public static final OptionalParameter<Integer> YARN_HEARTBEAT_INTERVAL = new OptionalParameter<>();
+  /**
+   * @see org.apache.reef.runtime.yarn.driver.parameters.YarnFederation.class
+   */
+  public static final OptionalParameter<Boolean> YARN_FEDERATION = new OptionalParameter<>();
```

Comment:
Please add links to the relevant JIRAs and a description as to what this is used for.